packages/client/hmi-client/src/types/Types.ts

```diff
@@ -801,9 +801,9 @@ export interface ModelUnit {
 }
 
 export interface GroundedSemantic {
-    grounding?: ModelGrounding;
     id: string;
     name?: string;
+    grounding?: ModelGrounding;
```

Comment:
This worryingly seems to change the order of items in this interface on every PR now?